client/layout/index.jsx

```diff
@@ -143,6 +143,7 @@ Layout = React.createClass( {
 		return (
 			<div className={ sectionClass }>
 				<DocumentHead />
+				<QuerySites allSites />
```

Comment:
Yes, that makes sense. Sooner or later all logged in sections need sites list ...